src/language/LanguageManager.js

```diff
@@ -263,7 +285,7 @@ define(function (require, exports, module) {
      *
      * @return {Language} The language for the provided file type or the fallback language
      */
-    function getLanguageForPath(path, ignoreOverride) {
+    function getLanguageForPath(path, ignoreOverride, rawText) {
```

Comment:
yea that's true ... my bad.  It is not really appropriate to pass content through `getLanguageForPath()`.  I think `getLanguageForContent()` should be kept for contents and `getLanguageForPath()` for path only, just like other `getLanguageFor*()` methods.  Do only one thing in a method.